public/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--where my maps are coming from-->
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Brook Yelpington</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+    integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+    crossorigin="" />
+  <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+    integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+    crossorigin=""></script>
+  <script src="https://kit.fontawesome.com/9a9c81b339.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="main.css" />
+</head>
+
+<!--Body of page-->
+
+<body>
+
+  <header class="header">
+    <img id="fish" src="../images/headerGraphics/headerGraphics.svg">
+  </header>
+
+  <div class="container">
+
+    <div id="mapid">
+    </div>
+    
+    <a href='restaurant' class='restBox'>A Menu for Menus
+      <ul id="restaurantList"></ul>
+    </a>
```

Comment:
Here, doing this will make your entire ```ul``` a link to the path in the href. If you put the anchor tag inside of the unordered list, then that would do what you are trying to achieve.
```html
<ul id="restaurantList">
  <a href='restaurant' class='restBox'>A Menu for Menus</a>
</ul>
```